src/nyc_trees/apps/survey/templates/survey/partials/map_another_popup.html

```diff
@@ -2,15 +2,23 @@
     <div class="modal-dialog">
         <div class="modal-content">
             <div class="modal-body">
-                <div>Map another block?</div>
+                <div class="hidden" data-class="map-another-possible">Map another block?</div>
+                <div class="hidden" data-class="map-another-impossible">You have completed mapping your reservations!</div>
             </div>
             <div class="modal-footer">
-                <a id="map-another-no" href="{% url 'progress_page' %}" class="btn btn-default">
-                    I'm Done For Now
-                </a>
-                <a id="map-another-yes" href="{% url 'survey' %}" class="btn btn-primary">
-                    Yes!
+                <a id="map-another-no"
+                   class="btn btn-default"
+                   data-class="map-another-possible"
+                   href="{% url 'progress_page' %}">I'm Done For Now</span>
+                <a id="map-another-no"
+                   class="btn btn-default"
+                   data-class="map-another-impossible"
+                   href="{% url 'progress_page' %}">OK</span>
```

Comment:
@dboyer @jfrankl  This needs content review. How does this look?

![screenshot from 2015-03-10 18 59 26](https://cloud.githubusercontent.com/assets/1699455/6587192/9baf9358-c757-11e4-8e29-8a2f331649f0.png)